app/controllers/pages_controller.rb

```diff
@@ -16,4 +16,24 @@ def trn_found
   def start
     session[:form_complete] = false
   end
+
+  def performance # rubocop:disable Metrics/AbcSize
+    trn_requests = TrnRequest.order(created_at: :desc).where(created_at: 1.week.ago.beginning_of_day..Time.zone.now)
+
+    trn_requests_from_days_ago = (0..6).map { |n| trn_requests.where(created_at: n.days.ago.all_day) }
+
+    @requests_over_last_7_days = trn_requests.count
+
+    @live_service_data = [%w[Date Requests]]
+    @live_service_data +=
+      trn_requests_from_days_ago.map.with_index { |reqs, i| [i.days.ago.strftime('%d %B'), reqs.count] }
+
+    @trns_found = trn_requests.where.not(trn: nil).count
+
+    @submission_data = [['Date', 'TRNs found', 'Zendesk tickets opened']]
+    @submission_data +=
+      trn_requests_from_days_ago.map.with_index do |reqs, i|
+        [i.days.ago.strftime('%d %B'), reqs.where.not(trn: nil).count, reqs.where.not(zendesk_ticket_id: nil).count]
+      end
+  end
```

Comment:
Doing a lot of lookups in this method, here's a screenshot of the requests:

![image](https://user-images.githubusercontent.com/1650875/164272983-c22ab442-b721-4bcc-ad30-a134ec65ca80.png)

There's 20+ database queries here. They're fast enough because they're all just retrieving counts. But is there a way to make them faster by using something like eager loading? I'm not super well versed in this.